catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/transformer/TransformerManager.java

```diff
@@ -99,6 +109,22 @@ public TransformerManager(List<ServiceReference> serviceReferences) {
         return null;
     }
 
+    public Map<String, Object> getRelatedTransformerProperties(String property, String value) {
+        Map<String, Object> map = new HashMap<>();
+        for (ServiceReference serviceRef : serviceRefs) {
+            Object propertyObject = serviceRef.getProperty(property);
+            if (propertyObject != null && propertyObject instanceof String) {
+                if (value.equals(propertyObject)) {
```

Comment:
what if there is more than one service that satisfies this? Multiple Transformers have `mime-type=text/xml`, now I'll only ever get the properties associated with one.